front/src/modules/users/components/FilterDropdownUserSearchSelect.tsx

```diff
@@ -40,6 +42,9 @@ export function FilterDropdownUserSearchSelect({
     }),
     searchFilter: filterDropdownSearchInput,
   });
+  const setHotkeyScope = useSetHotkeyScope();
+
+  setHotkeyScope(RelationPickerHotkeyScope.RelationPicker);
```

Comment:
This is dangerous because it is called at every render of the component and it might potentially override another setHotkeyScope call that would be made by another mounted component. 

There are 2 solutions for this case : 
- Put this in a useEffect that will only be triggered during component mount
- Call this in the action that is making this component mounting, usually in a button.

The second option is almost always the best option because you have full control on what is done and the insurance that it will be called only once, thus preventing any unexpected call from a re-render.

Reply:
If you have multiple places where this component can be mounted, you can centralized the call in a hook that will be used to open this component like useOpenUserSearch()

Reply:
My bad sure this shouldn't happen ... 
I have moved it temporarily to a useEffect - im not sure where I could place such a call in the filter logic as the components are mounted dynamically (e.g. in tasks-filters.tsx the entitySelectComponent ... ). Do you have an idea ?